packages/mux-player/REFERENCE.md

```diff
@@ -1,110 +1,144 @@
-| `poster`                      | `string` (URL)                                                      | Assigns a poster image URL. Will use the automatically generated poster based on your playback-id by default.                                                                                                                                                                                                                                                                                    | Derived       |
-| `beacon-collection-domain`    | `string` (domain name)                                              | Assigns a custom domain to be used for Mux Data collection.                                                                                                                                                                                                                                                                                                                                      | N/A           |
-| `custom-domain`               | `string` (domain name)                                              | Assigns a custom domain to be used for Mux Video.                                                                                                                                                                                                                                                                                                                                                | N/A           |
-| `nohotkeys`                   | `boolean`                                                           | Toggles keyboard shortcut (hot keys) support when focus in inside the player                                                                                                                                                                                                                                                                                                                     | `false`       |
-| `hotkeys`                     | `string`                                                            | Offers a way to disable certain keyboard shortcuts                                                                                                                                                                                                                                                                                                                                               | N/A           |
```

Comment:
should we start using https://prettier.io/docs/en/ignore.html#range-ignore for markdown tables?
for Media Chrome too. there's no way to see the real changes anymore w/ Prettier on

Reply:
Maybe. Let's discuss this in the next eng sync. Ideally I'd like these to be generated from in-code docs/types if possible or some other way of treating this as "data" instead of our current strategy (too prone to human error, adds extra work that robots could do, etc.).

Reply:
Fwiw, for this particular case, the order changed and some of the table column counts changed anyway, so the diff would still not be especially handy. Not sure if you caught it, but the description for the PR suggests some easier ways to review.